src/cubical/Cof.mli

```diff
@@ -34,3 +34,9 @@ val top : ('a, 'v) cof
 
 (** The boundary [∂r] of a dimension [r : 𝕀] is the disjunction [r=0 ∨ r=1] *)
 val boundary : dim0:'r -> dim1:'r -> 'r -> ('r, 'v) cof
+
+val complexity_cof : ('r, 'a) cof -> int
```

Comment:
What's this operator for?

Reply:
This is to keep "simpler" cofibrations instead of more complicated ones. (Not used in the current codebase.)